### PR TITLE
Cookie stack trace

### PR DESCRIPTION
**Pull request by jonathanKingston**
## Description:

Goes with https://github.com/duckduckgo/content-scope-scripts/pull/349
See https://app.asana.com/0/0/1204257980027287/f

## Automated tests:
- [ ] Unit tests
- [ ] Integration tests

###### Reviewer Checklist:
- [ ] **Ensure the PR solves the problem**
- [ ] **Review every line of code**
- [ ] **Ensure the PR does no harm by testing the changes thoroughly**
- [ ] **Get help if you're uncomfortable with any of the above!**
- [ ] Determine if there are any quick wins that improve the implementation

###### PR Author Checklist:
- [ ] Get advice or leverage existing code
- [ ] Agree on technical approach with reviewer (if the changes are nuanced)
- [ ] Ensure that there is a testing strategy (and documented non-automated tests)
- [ ] Ensure there is a documented monitoring strategy (if necessary)
- [ ] Consider systems implications

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -95,6 +95,7 @@ npm:
 
 ## clean: Clear the builds and temporary files.
 clean:
+	rm -rf shared/data/bundled/tracker-lookup.json
```
Comment (sammacbeth) [inline Makefile:98]: If we're committing this to source control, why are we deleting on clean?

Either this is source control, then we don't need these makefile changes. Then we update it with `npm run bundle-config`.

Alternatively, it's not in source control, and generated with the build and not by `bundle-config`.

File: shared/data/etags.json [hunk 2]
```diff
@@ -1 +1 @@
-{"config-etag":"W/\"260cf6ba6d4ac2cf2e2a5ef94c9edcdc\""}
\ No newline at end of file
+{"config-etag":"W/\"f1fa2ea9d9d27db1c7a68d7a81f6242d\""}
```
Comment (sammacbeth) [inline shared/data/etags.json:1]: Why has the etag changed when the content has not? Can we just drop these data changes from this diff?

**Review by sammacbeth — CHANGES_REQUESTED** [review]
We're mixing two methods of updating tracker-lookup here. We should pick one. IMO it's file to continue with `tracker-lookup.json` in version control, meaning you can drop the makefile changes here (just remove the `shared/data/bundled/tracker-lookup.json` target).

File: Makefile (hunk 1 above)
Comment (jonathanKingston) [reply Makefile:98]: As I was getting the following with Make without it when running `npm run bundle-confiug`:
```

> bundle-config
> node scripts/bundleConfig.mjs && make shared/data/bundled/tracker-lookup.json

Config imported successfully
make: `shared/data/bundled/tracker-lookup.json' is up to date.
```

File: shared/data/etags.json (hunk 2 above)
Comment (jonathanKingston) [reply shared/data/etags.json:1]: Me messing about, removed.

**Comment by jonathanKingston** [comment]
> We're mixing two methods of updating tracker-lookup here. We should pick one. IMO it's file to continue with `tracker-lookup.json` in version control, meaning you can drop the makefile changes here (just remove the `shared/data/bundled/tracker-lookup.json` target).

Ok, moved just to be npm only.

**Review by sammacbeth — APPROVED** [review]

**Pull request merged**